arroyo/processing/processor.py

```diff
@@ -142,9 +142,10 @@ def __init__(
 
         self.__message: Optional[BrokerValue[TStrategyPayload]] = None
 
+        self.__backpressure_timestamp: Optional[float] = None
         # If the consumer is in the paused state, this is when the last call to
-        # ``pause`` occurred or the time the pause metric was last recorded.
-        self.__paused_timestamp: Optional[float] = None
+        # ``pause`` occurred.
```

Comment:
Nit: This comment does not align with how self.__is_paused is used now. Can you update the comment?

Reply:
Fixed